src/components/Loader/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './Loader';
```

Comment:
Consider using a clear way to import and a clear way to export to maintain consistency. Instead of exporting directly from the import, you can import the component first and then export it. 😊